owncloudData/src/main/java/com/owncloud/android/data/webfinger/datasources/RemoteWebFingerDatasource.kt

```diff
@@ -17,6 +17,19 @@
  */
 package com.owncloud.android.data.webfinger.datasources
 
-interface WebfingerRemoteDatasource {
-    fun getJRDFromWebFingerHost(lookupServer: String, username: String): String
+import com.owncloud.android.domain.webfinger.model.WebFingerRel
+
+interface RemoteWebFingerDatasource {
+    fun getInstancesFromWebFinger(
+        lookupServer: String,
+        rel: WebFingerRel,
+        username: String
+    ): List<String>
+
+    fun getInstancesFromAuthenticatedWebfinger(
```

Comment:
```suggestion
    fun getInstancesFromAuthenticatedWebFinger(
```
This should be renamed as well 👍